addons/payment/views/payment_form_templates.xml

```diff
@@ -408,37 +408,70 @@
 
     <template id="payment.no_pms_available_warning">
         <div class="alert alert-warning mt-2">
-            <strong>No suitable payment option could be found.</strong>
-            <!-- === Availability report button (for debug) === -->
-            <t t-call="payment.availability_report_button"/>
+            <strong>No payment method available for your order.</strong>
+            <!-- === Availability report button === -->
             <br/>
-            If you believe that it is an error, please contact the website administrator.
+            <t t-if="request.env.is_admin()">
+                <p>You didn't configure any payment provider.</p>
+            </t>
+            <t t-else="">
+                <t t-if="has_active_payment_provider">
+                    <p>If you believe that it is an error, please contact the website administrator.</p>
+                </t>
+                <t t-else="">
+                    <p>No payment method is available for
+                        <t t-out="request.env.company.country_id.name"/>,
+                        <t t-out="request.env.company.currency_id.name"/>.
+                    </p>
```

Comment:
I think this part should be visible to admin if payment provider is installed :thinking:

Reply:
Yes, It will be visible to the admin when payment provider is installed.
Made some changes in the condition :+1:

Reply:
oops I forgot Also
we need this for both :thinking:

Reply:
No, we need that message only for admin.
when a payment provider is installed but we don't have the payment method active.